packages/bem-react-scripts/scripts/build.js

```diff
@@ -66,134 +76,19 @@ function printErrors(summary, errors) {
 function build(previousFileSizes) {
   console.log('Creating an optimized production build...');
 
-  let compiler;
   try {
-    compiler = webpack(config);
+    parallelWebpack.run(
+        configPath,
+        {
+            watch: webpackArgv.watch,
+            maxRetries: webpackArgv.maxRetries,
+            stats: webpackArgv.stats
+        }
+    );
   } catch (err) {
     printErrors('Failed to compile.', [err]);
     process.exit(1);
   }
-
-  compiler.run((err, stats) => {
-    if (err) {
-      printErrors('Failed to compile.', [err]);
-      process.exit(1);
-    }
```

Comment:
We delete a lot of colourful messages :(

Reply:
parallel-webpack gives a lot of information